twoaxistracking/twoaxistrackerfield.py

```diff
@@ -73,6 +73,11 @@ def __init__(self, total_collector_geometry, active_collector_geometry,
         self.min_tracker_spacing = \
             layout._calculate_min_tracker_spacing(self.total_collector_geometry)
 
+        # Ensure that the total collector area contains the active areas
+        if self.total_collector_geometry.contains(self.active_collector_geometry) is False:
+            raise ValueError('The total collector geometry does not completely'
+                             ' enclose the active collector geometry.')
+
```

Comment:
@kanderso-nrel I per your suggestion, a ValueError will now be raised if the total collector geometry does not completely enclose the active geometries.